pkg/daemon/ceph/agent/agent.go

```diff
@@ -120,3 +121,26 @@ func (a *Agent) Run() error {
 		}
 	}
 }
+
+// In 1.11 and newer there is a timing issue loading flex drivers.
+// See https://github.com/rook/rook/issues/1501 and https://github.com/kubernetes/kubernetes/issues/60694
+// With this loop we constantly make sure the flex drivers are all loaded.
+func periodicallyRefreshFlexDrivers(driverName string, stopCh chan struct{}) {
+	waitTime := 2 * time.Minute
+	for {
+		logger.Debugf("waiting %s before refreshing flex", waitTime.String())
+		select {
+		case <-time.After(waitTime):
+			flexvolume.TouchFlexDrivers(flexvolume.FlexvolumeVendor, driverName)
+
+			// increase the wait time after the first few times we refresh
+			if waitTime < 32*time.Minute {
```

Comment:
I think it would be good mention that the "maximum" wait time is 32 minutes in the comment. :slightly_smiling_face: